CorpusSearch/ClientApp/src/routes/BitPlayer.tsx

```diff
@@ -47,42 +50,73 @@ function Video({vId="rLEBp8R1_XA"}) {
         setDur(the_player.getDuration())
     }
     const onSChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-        setStartTime(e.target.value)
+        setStartTime(constrained_val(parseFloat(e.target.value)) )
     }
     const onEChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-        setEndTime(e.target.value)
+        setEndTime(constrained_val(parseFloat(e.target.value)) )
     }
-    const onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const onIdChange = (e: React.ChangeEvent<HTMLInputElement>) => {
         setVideoId(e.target.value)
     }
     const restart = () => {
-        r_player.current.seekTo(parseFloat(startTime))
+        r_player.current.seekTo(startTime)
         r_player.current.playVideo()
     }
     const onClick = (/*e :MouseEvent<HTMLButtonElement>*/) => {
         setKtime(r_player.current.getCurrentTime())
         restart()
     }
-
     /* eslint-enable */
+    const constrained_val = (n: number) => {
```

Comment:
I'd be tempted to return null here

Reply:
Maybe but 
<img width="1094" alt="NullA" src="https://github.com/david-allison/manx-corpus-search/assets/47546076/bd88aded-d31f-457c-a681-ec25efa84be5">